mapper/orm/src/main/java/org/hibernate/search/mapper/orm/mapping/impl/HibernateOrmMapping.java

```diff
@@ -182,22 +183,22 @@ public CompletableFuture<?> start(MappingStartContext context) {
 
 		// Schema management
 		PojoScopeSchemaManager schemaManager = scope.schemaManagerDelegate();
-		return schemaManagementListener.onStart( context, schemaManager )
+		return schemaManagementListener.onStart( context, schemaManager, operationSubmitter )
```

Comment:
It's really okay to block if queues are full on startup/shutdown, IMO. Who would want to fail if a queue gets full on startup? I'd expect everyone to want to wait until startup finishes.